src/panoramic/cli/config/storage.py

```diff
@@ -0,0 +1,29 @@
+import os
+from typing import Any, Dict
+
+from panoramic.cli.file_utils import read_yaml, write_yaml
+from panoramic.cli.paths import Paths
+
+
+def update_config(section: str, data: Dict[str, Any]) -> None:
+    if section != '':
```

Comment:
Is `''` a special case that we expect to happen? If yes, can we use `None` instead? If not, I'd remove this check

Reply:
Removed